crates/cairo-addons/src/vm/runner.rs

```diff
@@ -396,6 +396,34 @@ except Exception as e:
         Ok(())
     }
 
+    fn verify_squashed_dicts(&mut self) -> PyResult<()> {
+        let dict_manager_ref = self
+            .inner
+            .exec_scopes
+            .get_dict_manager()
+            .map_err(|e| PyErr::new::<pyo3::exceptions::PyRuntimeError, _>(e.to_string()))?;
+        let dict_manager = dict_manager_ref.borrow();
+        let all_trackers = dict_manager.trackers.values().collect::<Vec<_>>();
+        let mut unsquashed_trackers = Vec::new();
+        for tracker in all_trackers {
+            if !tracker.is_squashed {
+                unsquashed_trackers.push(tracker);
+            }
+        }
```

Comment:
rust things but you could probably avoid the loop by creating directly the unsquashed_trackers like `.values().filter().collect()`